src/styles/calendar.scss

```diff
@@ -0,0 +1,44 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  width: $calendar-width;
+  gap: 1px;
+
+  @each $day, $i in $start-day {
+    &--start-day-#{$day} .calendar__day:first-child {
+      margin-left: calc(#{$i} * 100px + #{$i * 1px});
+    }
+  }
+
+  @for $numberday from 28 to 31 {
+    &--month-length-#{$numberday} :nth-child(#{$numberday + 1}) {
```

Comment:
check logic in this selector.
now it not works correctly and is the reason of failed test.

https://github.com/user-attachments/assets/681405cb-c23d-49e6-a365-2080278bfdd5